google/cloud/bigtable/internal/bulk_mutator.h

```diff
@@ -92,9 +89,17 @@ class BulkMutatorState {
      * request provided by the application.
      */
     int original_index;
-    google::cloud::Idempotency idempotency;
+    Idempotency idempotency;
     /// Set to `false` if the result is unknown.
     bool has_mutation_result;
+    /**
+     * The last known status for this annotation.
+     *
+     * If the final stream attempt has failing mutations, but ends with an OK
+     * status, we return a `FailedMutation` made from `original_index` and
+     * `status`. The value is meaningless if `has_mutation_result` is false.
+     */
+    Status const status;
```

Comment:
The `const` makes the `struct` unassignable, which is weird, and (I'd say) isn't worth it.  Should other fields be `const` too, for example.

Reply:
`has_mutation_result` can't be `const`. The other fields could be. I will take your suggestion and drop the `const` here.